README.md

```diff
@@ -54,7 +56,7 @@ use iocuddle::*;
 
 let mut file = std::fs::File::open("/dev/tty").unwrap_or_else(|_| std::process::exit(0));
 
-// This is the simplist ioctl call. The request number is provided via the
+// This is the simplest ioctl call. The request number is provided via the
```

Comment:
@mbestavros Please fix this in `src/lib.rs` too. It would be awesome if we could figure out how to import the `README.md` into the `src/lib.rs` or vice versa.